docker/nginx/nginx.conf

```diff
@@ -0,0 +1,88 @@
+#ssl_certificate /etc/letsencrypt/live/myopenhab.org/fullchain.pem;
+#ssl_certificate_key /etc/letsencrypt/live/myopenhab.org/privkey.pem;
+worker_processes 4;
+
+events { worker_connections 1024; }
+
+http {
+
+upstream app_server {
+        server app-1:3000 fail_timeout=5s max_fails=5;
+}
+
+server {
+    listen 8080 default_server;
```

Comment:
the target is to use only one nginx config from oh cloud. This nginx config will be removed completely. Yes 8080 was a relict and will be 80.